src/content/changes/highlights/2022-09-15.md

```diff
@@ -0,0 +1,32 @@
+---
+date: 2022-09-15T10:00:00
+title: Highlights for the week ending September 15, 2022
+changes_categories:
+  - Highlights
+owner:
+  - https://github.com/orgs/giantswarm/teams/sig-product
+---
+
+## General
+
+
+
+## User interfaces
+
+In [kubectl-gs](https://github.com/giantswarm/kubectl-gs/releases/tag/v2.22.0) we have deprecated some flags in three commands, to avoid name conflicts with global flags (which are usable in all commands). You will see a warning when using the deprecated flags. This affects the following commands:
+
+- `kubectl gs template app`: flag `--target-namespace` replaces `--namespace`, `--cluster-name` replaces `--cluster`.
+- `kubectl gs template catalog`: flag `--target-namespace` replaces `--namespace`.
+- `kubectl gs gitops add app`: flag `--target-namespace` replaces `--namespace`.
+
+Please move any scripts to the new flags, as the old flags will change their meaning with the next major release of kubectl-gs.
+
+kubectl-gs now also provides a command `get organizations`, which is a way for all management API users to list the organizations they have access to. `kubectl get organizations` in contrast requires admin permissions.
+
+## Apps
+
+- [trivy-app](https://github.com/giantswarm/trivy-app) version [v0.5.0](https://github.com/giantswarm/trivy-app/blob/main/CHANGELOG.md#050---2022-09-09) updates to Trivy version `0.29.2`, introducing support for Trivy modules. The `spring4shell` module is installed by default.
+- [prometheus-remotewrite](https://github.com/giantswarm/prometheus-remotewrite) version [v0.1.0](https://github.com/giantswarm/prometheus-remotewrite/blob/main/CHANGELOG.md#010---2022-07-13) With this release we are enabling Prometheus to replicate its data into 3rd party systems like Grafana Cloud or even another Prometheus using its [remote APIs](https://prometheus.io/blog/2019/10/10/remote-read-meets-streaming/#remote-apis). More details can be found in the [documentation](https://docs.giantswarm.io/ui-api/observability/prometheus-remotewrite/)
+- [cert-manager-app](https://github.com/giantswarm/cert-manager-app) version [v.2.16.0](https://github.com/giantswarm/cert-manager-app/blob/master/CHANGELOG.md#2160---2022-09-12) Before you upgrade to this release, make sure to read the [Upgrading from v1.7 to v1.8](https://cert-manager.io/docs/installation/upgrading/upgrading-1.7-1.8/) document. Upgrade to upstream image [v1.8.2](https://github.com/jetstack/cert-manager/releases/tag/v1.8.2)
```

Comment:
- [gatling-app](https://github.com/giantswarm/gatling-app) version [v2.0.1](https://github.com/giantswarm/gatling-app/blob/master/CHANGELOG.md#201---2022-09-14) Update Gatling to v3.8.4.
- [gatling-app](https://github.com/giantswarm/gatling-app) version [v2.0.0](https://github.com/giantswarm/gatling-app/blob/master/CHANGELOG.md#200---2022-09-14) Complete rework based on fresh Helm chart. Following app related keys of the values.yaml changed: `job.* moved to top-level.` `simulation.configMap.name -> simulation.configmap`. `simulation.filename -> simulation.file`. `simulation.name -> simulation.class`.
- [gatling-app](https://github.com/giantswarm/gatling-app) version [v1.1.0](https://github.com/giantswarm/gatling-app/blob/master/CHANGELOG.md#110---2022-09-13) Update Gatling to v3.8.3.